docs/dev-guide/tscript/debugging.mdx

```diff
@@ -190,13 +190,20 @@ The two main sources of dev-prod discrepancies are in bundling and connecting.
 
 #### Production bundling
 
+You may experience your Client sending stripped names as the Workflow "Type" when scheduling a Workflow.
 If you are getting errors like this:
 
 ```
 Error: 3 INVALID_ARGUMENT: WorkflowType is not set on request.
 ```
 
-This is due to your bundler stripping out Workflow function names, which we rely on to set the "Workflow Type" in Temporal. Turn it off and it should work.
+Or, if you are seeing stripped-down Workflow types in the Temporal Service's Web UI like this:
+
+![Temporal Web UI showing stripped "Workflow Type" entries, in this case the single letter 's'](/img/stripped_workflow_types_in_webui.png)
+
+It can be due to your bundler stripping out Workflow function names.
+Temporal relies on those names to set the "Workflow Type" in the Service Web UI.
+Disable that by configuring webpack (in webpack.config.js) and it should work.
```

Comment:
```suggestion
To prevent the build process from removing Workflow function names,  modify the webpack configuration file ( `webpack.config.js`) by setting `keep_fnames` to `true` in the `TerserPlugin` configuration section:
```